domains/dave9123.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "dave9123",
+        "email": "dave.turmawan@outlook.com"
+    },
+    "record": {
+        "CNAME": "dave9123.github.io"
```

Comment:
CNAME still redirects to this domain, which would redirect back to your GitHub page, creating a loop.